pom.xml

```diff
@@ -468,11 +475,6 @@
         <version>${google.flogger.version}</version>
         <scope>runtime</scope>
       </dependency>
-      <dependency>
```

Comment:
why we need to remove this? This is getting added indirectly from GCS Client library?

Reply:
> This is getting added indirectly from GCS Client library?

Yes.